docs/formats.md

```diff
@@ -87,6 +88,12 @@ To decode quantized height value:
 
   `255 - bisect.bisect_left(HEIGHT_TABLE, h)`
 
+## Buffered Terrarium and Normal
```

Comment:
This should move to the Use Service page, which has other URL related changes. You'll also need to give example URLs for each.